LastWork(newWork)/Views/common/common.js

```diff
@@ -1,20 +1,26 @@
-const Footer = function (option) {
-    return `
+interviewApp.Footer = {
+    init() {
+        document.querySelector('#footer').innerHTML =
+    `
     <section class="footer_section">
     <footer>
         <p>Copyright@Softvision 2017</p>
     </footer>		
     </section>	
     `;
+    },
+    destroy() {
+        document.querySelector('#footer').innerHTML = '';
+    },
 };
 
-const NAV = function (option) {
-    var evalSelected;
-    var newEvalSelected;
-    if (option === 'Eval') {evalSelected = 'button_focus'; }
-    else if (option === 'newEval')
-    { newEvalSelected = 'button_focus'; }
-    return `
+interviewApp.NAV = {
+    init(option) {
+        let evalSelected;
+        let newEvalSelected;
+        if (option === 'Eval') { evalSelected = 'button_focus'; } else if (option === 'newEval') { newEvalSelected = 'button_focus'; }
```

Comment:
I find this particular line of code to be very interesting. What are you trying to achieve here?